contrib/reaction/files/reaction.yml

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
you could also install this file from ./app/example.yml afaik

Reply:
I do, reaction.yml is just that file plus a header with more clarifications

Reply:
the default config hard depends on an iptables setup, so just in case someone running nftables or firewalld installs the package and fails to start it, those will be a great indication of what went wrong

Reply:
well i suppose since it doesn't change anything functionally might as well delete it.